src/SwedbankPay/Api/Client/Client.php

```diff
@@ -62,8 +62,25 @@ public function __construct($data = [])
 
         // Set version
         if (!$this->offsetExists(self::CLIENT_VERSION)) {
-            $data = json_decode(file_get_contents(__DIR__ . '/../../../../composer.json'), true);
-            $this->setVersion($data['version']);
+            $version = false;
+
+            if (defined('VERSION')) {
+                $version = VERSION;
+            } elseif (getenv("VERSION") !== false) {
+                $version = getenv("VERSION");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function getenv() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5251412)